packages/kit/src/vite/dev/index.js

```diff
@@ -32,6 +32,11 @@ export async function dev(vite, vite_config, svelte_config, illegal_imports) {
 
 	sync.init(svelte_config, vite_config.mode);
 
+	/** @type {Partial<import('types').Hooks>} */
+	const user_hooks = resolve_entry(svelte_config.kit.files.hooks)
+		? await vite.ssrLoadModule(`/${svelte_config.kit.files.hooks}`)
+		: {};
+
```

Comment:
i think this needs to happen after `set_private_env` and `set_public_env`, no? otherwise anything that references dynamic env vars in hooks would fail, i think